templates/actors/character-sheet.hbs

```diff
@@ -52,15 +52,16 @@
             {{!-- Header Attributes --}}
             <ul class="attributes flexrow">
                 <li class="attribute health">
-                    <h4 class="attribute-name box-title">{{ localize "DND5E.HitPoints" }}</h4>
+                    <h4 class="attribute-name box-title">{{localize "DND5E.HitPoints"}}</h4>
+                    <a class="config-button" data-action="hit-points"
+                        data-tooltip="{{localize 'DND5E.HitPointsConfig'}}"><i class="fas fa-cog"></i></a>
                     <div class="attribute-value multiple">
                         <input name="system.attributes.hp.value" type="text"
```

Comment:
Would be good to convert `<input type="text" data-dtype="Number">` into `<input type="number">`

Reply:
These were converted back to `text` because a bug in Chrome prevents players from entering delta values (`+10` to add 10 rather then set to 10).